lib/modem_info/modem_info.c

```diff
@@ -720,6 +733,144 @@ int modem_info_rsrp_register(rsrp_cb_t cb)
 	return 0;
 }
 
+/*
+ * Obtain and copy out a string value from a response to an AT command.
+ * One buffer is used internally and dimensioned for the known maximum size of
+ * the value, while the other is the user provided target buffer.
+ *
+ * cmd     Command to execute
+ * fmt     Format string by which to extract 1 string from response
+ * tmp_buf Buffer to hold response from nrf_modem_scanf
+ * out_buf Buffer to finally write the string to
+ */
+int at_scanf_string(const char *cmd, const char *fmt, char *tmp_buf,
```

Comment:
I don't fully understand why we need the wrapper functions for `nrf_modem_at_scanf`. Is it to map return codes so `0` is success?